src/main/java/com/cloudogu/smeagol/wiki/infrastructure/ScmGitPageRepository.java

```diff
@@ -15,6 +15,8 @@
 import java.time.Instant;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.time.Instant'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)